### PR TITLE
AC New Test - LPD-29404 | master

### PR DESCRIPTION
**Pull request by ViniLopes20**
https://liferay.atlassian.net/browse/LPD-29404

### CODE REVIEW

**Comment by marcosapmf** [comment]
ci:report:28576492

**Comment by marcosapmf** [comment]
ci:report:29207888

**Comment by interaminense** [comment]
ci:test:analytics-cloud-acceptance

File: modules/test/playwright/tests/osb-faro-web/utils/selectors.ts [hunk 1]
```diff
@@ -2,7 +2,7 @@
  * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
  * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
  */
-export enum CardSelector {
+export enum cardSelector {
```
Comment (interaminense) [inline modules/test/playwright/tests/osb-faro-web/utils/selectors.ts:5]: Hey @ViniLopes20, enum should start with the first letter uppercase.

**Review by interaminense — CHANGES_REQUESTED** [review]

**Comment by marcosapmf** [comment]
@ViniLopes20 
resent to https://github.com/liferay-ac/liferay-portal/pull/1663

**Pull request closed without merging**